descriptor_test.go

```diff
@@ -0,0 +1,139 @@
+package oras_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"oras.land/oras-go/v2"
+)
+
+func TestGenerateDescriptor(t *testing.T) {
+	type args struct {
+		content   []byte
+		mediaType string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    ocispec.Descriptor
+		wantErr bool
+	}{
+		{
+			name: "foo descriptor",
+			args: args{[]byte("foo"), "example media type"},
```

Comment:
nit: We can still declare the content as a variable at the beginning of the test function.

Reply:
But we have 3 contents (foo, bar, foo). Do we need to declare them all? I think writing out foo/bar is more handy.